tests/containers/openshift_setup.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Set-up OpenShift cluster on the host
+# There are some requirements that need to be pre-setup on the host:
```

Comment:
would be interesting to move this to a library? or will be like a previous step before launch any test on openshift?

Reply:
It will be executed as part of [Host Updates](https://openqa.suse.de/group_overview/377), once a week or so. I would keep it here as it won't be used by other tests.